web/themes/custom/tube_theme/templates/page.html.twig

```diff
@@ -0,0 +1,190 @@
+{#
```

Comment:
Were any changes made in this file? Generally speaking, when creating a new twig file (based on an existing one), you should commit the 'creation' of the file in one commit. Then you can commit edits to the file in subsequent commits. As-is, I can't tell what changes were made to the file, which makes it difficult to review

Reply:
Thank you for feedback. Page.html.twig file I only changed to container-fluid on the content area to make it full width. That is the only changes I made.

Reply:
Thank you letting me know. I’ll commit the file seprately next time.